Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿#if !WINDOWS_UWP
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System.Collections;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    class PointerTests 
+    {
+        [SetUp]
+        public void Setup()
+        {
+            PlayModeTestUtilities.Setup();
+            TestUtilities.PlayspaceToOriginLookingForward();
+
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            PlayModeTestUtilities.TearDown();
+        }
+
+        #region Tests
+
+        /// <summary>
+        /// Tests that right after being instantiated, the pointer's direction 
+        /// is in the same general direction as the forward direction of the camera
+        /// </summary>
+        /// <returns></returns>
+        /// 
+        [UnityTest]
+        public IEnumerator TestPointerDirectionMatchesCameraDirectionFirstFrame()
+        {
+            var inputSystem = PlayModeTestUtilities.GetInputSystem();
+
+            // Raise the hand
+            var rightHand = new TestHand(Handedness.Right);
+
+            //Position 1 show hand
+            Vector3 initialPos = new Vector3(0.01f, 0.1f, 0.5f);
+            yield return rightHand.Show(initialPos);
+
+            //return first hand controller that is right and source type hand
+            var handController = inputSystem.DetectedControllers.First(x => x.ControllerHandedness == Utilities.Handedness.Right && x.InputSource.SourceType == InputSourceType.Hand);
+            Assert.IsNotNull(handController);
+
+            // Get the line pointer from the hand controller
+            var linePointer = handController.InputSource.Pointers.First(x => x is LinePointer);
+            Assert.IsNotNull(linePointer);
+
+            // Take dot product of camera forward and line pointer
+            float dot = Vector3.Dot(linePointer.Rays[0].Direction, Camera.main.transform.forward);
+
+            // Making sure the pointer is pointing in the same general direction as the camera
+            // A dot of 1 means the pointer and camera forward point in the exact same direction 
+            // but greater than 0.5 is same general direction
+            Assert.GreaterOrEqual(dot, 0.5f);
+
+            // Position 2
```

Comment:
It feels like all of the other tests here aren't testing the name of the test (i.e. they're not actually about the first frame) - what's the intent of the rest of the test here (also something that I wasn't sure about - I would assume that the direction of the hand is independent of it's position, once its direction is set. If this isn't the case (i.e. based on what exists below), i guess maybe it's just interesting info for me to know.

Reply:
The direction of the line pointer is not independent from the position of the hand.  My understanding is that the line pointer direction changes based on the position of the hand relative to the camera.  But the range of motion for the hands in the input simulation is way larger than the actual range of motion of hands.

[Video - Line Pointer Direction Behavior](https://youtu.be/Fay6-mpgpW8)